codecov.yml

```diff
@@ -9,7 +9,7 @@ coverage:
     patch:
```

Comment:
Why are we moving the file out of `.github`?

Reply:
I think it was using the global YAML before, so we were getting checks, but it wasn't using this YAML's config

Reply:
It can go in .github too - https://docs.codecov.com/docs/codecov-yaml#can-i-name-the-file-codecovyml

Reply:
Ah ok nice, I'll move it back